data/test_moodbot/intent_ted_policy_config.yml

```diff
@@ -0,0 +1,23 @@
+language: en
+
+pipeline:
+  - name: WhitespaceTokenizer
+    intent_tokenization_flag: True
+  - name: RegexFeaturizer
+  - name: LexicalSyntacticFeaturizer
+  - name: CountVectorsFeaturizer
+  - name: CountVectorsFeaturizer
+    analyzer: char_wb
+    min_ngram: 1
+    max_ngram: 4
+  - name: "DIETClassifier"
+    entity_recognition: False
+    epochs: 1
+
+policies:
+  - name: MemoizationPolicy
+  - name: IntentTEDPolicy
```

Comment:
Should we have `TEDPolicy` too so that it can predict actions. Or is `MemoizationPolicy` enough for that?

Reply:
It's enough to have `IntentTEDPolicy` and `MemoizationPolicy`